db/schema.rb

```diff
@@ -1358,7 +1358,7 @@
     t.datetime "last_article_at", default: "2017-01-01 05:00:00"
     t.datetime "last_comment_at", default: "2017-01-01 05:00:00"
     t.datetime "last_followed_at"
-    t.datetime "last_moderation_notification", default: "2017-01-01 05:00:00"
+    t.datetime "last_moderation_notification", default: "2017-01-01 00:00:00"
```

Comment:
I think these schema.rb changes can be safely removed from the commit, it's probably something from your initial setup caused these changes (they're unrelated to your change above and not needed to render strikethrough)